modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -872,15 +872,18 @@
           "WORKFLOWS": {
             "ACTIONS": "Actions",
             "DETAILS": "Details",
-            "DESCRIPTION": "",
-            "MORE_INFO": "More Information & Actions",
-            "ID": "ID",
+            "DESCRIPTION": "Description",
+            "MORE_INFO": "More Information",
+            "ID": "WIID",
             "TYPE": "Type",
             "TITLE": "Title",
-            "STATE": "State",
-            "STARTED": "Started",
+            "STATUS": "Status",
+            "STARTED": "Started At",
```

Comment:
To be consistent with other tables, I suggest to not capitalize every word. Additionally, given only this string, you could not say what this field is for. E.g. it could also mean “Stared at location”. A better description (if you want to extend this) would be `Start date`

Reply:
I think it is not used anymore. Will check and remove the key if not used.

Note that there will be a date displayed as value so it is obvious that a date is meant.

Reply:
If we're looking for shorter we could just use `Started`, which is clear enough with the date and time next to it.